.gitignore

```diff
@@ -2,4 +2,6 @@
 __pycache__/
 venv/
 *.pyc
+/archive/
+*.test
```

Comment:
What are these? They don't look like common patterns to exclude. If it's something only you use, please find a different solution, e.g. change the global Git config.